wallet/src/util/WalletBackendAdapter.ts

```diff
@@ -402,7 +404,10 @@ export const makeWalletBridgeFromFollowers = (
       // If we ever add non 'set' amount purses that aren't in the vbank, it's
       // not currently possible to read their decimalPlaces, so this code
       // will need updating.
-      if (!Array.isArray(purse.balance.value)) {
+
+      const isCopyBag = endoIsCopyBag(purse.balance.value);
```

Comment:
that it comes from Endo isn't semantically meaningful. please omit from the name.

I think the bar should be high for renaming an import so I'd suggest keeping the imported name and using a different name for the boolean.

otherwise import the function as something meaningful like `isCopyBagValue`.